modules/ROOT/pages/6.6.2-release-notes.adoc

```diff
@@ -323,3 +323,15 @@ Values **x.1–x.4** are rounded to **x**.
 Values **x.5–x.9** are rounded to **x+1**.
 
 NOTE: If `color_cols` is set to a decimal value that would, after rounding, be an invalid value (for example, **0.1**), {productname} continues to act as it did previously. For invalid values, the `color_cols` option is treated as if it has not been set and reverts to displaying the color selection grid with the default five columns.
+
+
+=== An "Uncaught TypeError: Cannot read properties of null" error would sometimes be thrown when updating the content of a `streamContent: true` iframe dialog component
+//TINY-10128
+
+During rapid, intermittent, content updates in an iframe dialog component with `streamContent: true` set, an error — "Uncaught TypeError: Cannot read properties of null" — would sometimes be thrown.
+
+This caused an error when trying to scroll to the bottom of the iframe component.
+
+In {productname} 6.6.2 two changes were made: using a numerical value for `Window.scrollTo` rather than setting it to `body.scrollHeight`; and waiting for `load` before attempting to scroll.
```

Comment:
Due to a last-minute change, we no longer wait for `load` before attempting to scroll
```suggestion
{productname} 6.6.2 uses a numerical `y-coord` value for `Window.scrollTo` rather than the iframe body's scrollHeight. As a consequence, the error is no longer thrown, and maintaining scroll at the bottom of a `streamContent: true` iframe component works as expected.
```